backend/src/app/auth.py

```diff
@@ -11,6 +11,8 @@ def decorated_function(*args, **kwargs):
     auth_service = app.services.auth_service()
     (user, err) = auth_service.authenticate_request(request)
     if user:
+      g.user_id = user["uid"]
+      g.user_role = user["role"]
```

Comment:
Are you sure this is right? I see the following error in the pytest failure...

  File "/home/runner/work/housinghub/housinghub/backend/src/app/auth.py", line 14, in decorated_function
    g.user_id = user["uid"]
TypeError: 'User' object is not subscriptable

Reply:
Is it better to assume the created_by user's id to be passed from the front end? 
@thinkharderdev , what do you suggest?

Reply:
In general it's a good idea to have these fields come from a trusted source. Since the JWT token is controlled by the server and can't be altered by the client we know it is a valid value. If it's passed from the front-end then it can be manipulated and could be invalid